src/test/ui/macros/assert.stderr

```diff
@@ -0,0 +1,16 @@
+error: macro requires a boolean expression as an argument
+  --> $DIR/assert.rs:2:5
+   |
+LL |     assert!();  //~ ERROR requires a boolean expression
+   |     ^^^^^^^^^^ boolean expression required
```

Comment:
It'd be nice to point at the (empty) argument list, but this is already quite an improvement.